eng/pipelines/tools-repo-versioning.yml

```diff
@@ -0,0 +1,21 @@
+# Tag the (azure-sdk-tools) repo to create some form of versioning.
+trigger: none
+
+pr: none
+
+pool:
+  vmImage: windows-2019
+
+jobs:
+  - job: TagToolsRepo
+    displayName: Version azure-sdk-tools with Git Tag
+
+    steps:
+    - pwsh: |
+        $PushURL = "$(Build.Repository.Uri)".Replace("https://", "https://$(azuresdk-github-pat)@") + ".git"
+        $Tag = "$(Build.Repository.Name)".SubString("$(Build.Repository.Name)".IndexOf('/') + 1) + "_$(Build.BuildNumber)"
```

Comment:
I assume you are doing this because the name looks like "Azure/azure-sdk-tools"? Assuming the tag name accepts "/" I could live with just using the full name. However I'm also OK with what you have here as well but I'd suggest adding a comment above it telling why you are doing it.